src/wfs/src/test/java/org/geoserver/wfs/GetFeatureTest.java

```diff
@@ -375,7 +377,25 @@ public void testRequestDisabledResource() throws Exception {
                 doc);
         XMLAssert.assertXpathEvaluatesTo("typeName", "//ogc:ServiceException/@locator", doc);
     }
-    
+
+    /**
+     * Test that a request for a resource from a disabled store fails.
+     */
+    @Test
+    public void testRequestDisabledStore() throws Exception {
+        Catalog catalog = getCatalog();
+        StoreInfo store = catalog.getStoreByName("cdf", DataStoreInfo.class);
+        store.setEnabled(false);
```

Comment:
@aaime no luck at all: there is already a ```@Before resetFifteen()``` above in this file to ensure that every test gets a working ```cdf:Fifteen```. The new test method is not the first to modify the store. Seeing the ```@Before``` method, I knew that fixture clean up was not required.  :-)